test/controllers/task_controller_test.exs

```diff
@@ -35,10 +35,10 @@ defmodule CodeCorps.TaskControllerTest do
       json = conn |> get(path) |> json_response(200)
 
       ids =
-        json["data"]
-        |> Enum.map(&Map.get(&1, "id"))
-        |> Enum.map(&Integer.parse/1)
-        |> Enum.map(fn({id, _rem}) -> id end)
+      json["data"]
+      |> Enum.map(&Map.get(&1, "id"))
+      |> Enum.map(&Integer.parse/1)
+      |> Enum.map(fn({id, _rem}) -> id end)
```

Comment:
No, I recall checking it out a coupe years ago but didn't set it up for vim. Do you have settings that would play nice with the code-corps projects?